libs/eo/help/shell/src/lib/eo-introduction-page.component.ts

```diff
@@ -15,12 +15,251 @@
+          alt="Energy Origin graph of energy"
+        />
+        <h3>Hvad kan du bruge energioprindelse.dk til?</h3>
+        <eo-stack size="M">
+          <p>
+            energioprindelse.dk er hovedsageligt beregnet til, at du kan
+            <b>dokumentere</b>, hvor grøn du er. Du kan se alle
+            <b>dine data</b> og bruge dem i andre systemer.
+          </p>
+          <p>
+            Det betyder overordnet at du kan se din andel
+            <b>vedvarende energi, emissioner, forbrugs- og produktionsdata</b>.
+            Derudover kan du også se dine <b>certifikater</b>, både dem du selv
+            har produceret, og dem du har købt.
+          </p>
+          <p>Alle disse informationer kan du f.eks. bruge i din CSR-rapport.</p>
+        </eo-stack>
+      </eo-info-box>
+      <eo-info-box>
+        <h3>Grøn energi med EnergiOprindelse</h3>
+        <eo-stack size="M">
+          <p>
+            Energiforbrugere oplever et stigende behov for kendskab til deres
+            energis oprindelse, både for at kunne holde øje med deres
+            <b>forbrug</b>, og for at kunne dokumentere hvor grønne de er, dvs.
+            andelen af <b>vedvarende</b> energi og mængden af <b>emissioner</b>.
+          </p>
+          <p>
+            Indtil nu har det <b>ikke været muligt at vide</b>, om ens energi
+            faktisk ér grøn, når det forbruges. Det skyldes, at man i dag kun
+            har mulighed for at købe oprindelsesgarantier, der baseres på
+            <b>årsbasis</b>. Dvs. at man får en garanti for, at der findes nok
+            vedvarende energi i nettet for året, til at matche ens forbrug - men
+            det betyder desværre ikke, at energien faktisk var grøn på
+            forbrugstidspunktet.
+          </p>
+          <p>
+            Den usikkerhed gør EnergiOprindelse op med, for med platformen kan
+            energi, samt dens tilsvarende CO2-udledning, spores fra produktion
+            til forbrug helt ned på <b>timeniveau</b>.
+          </p>
+        </eo-stack>
+      </eo-info-box>
+      <eo-info-box>
+        <h3>Hvordan kan du blive grønnere?</h3>
+        <eo-stack size="M">
+          <p>
+            Helt overordnet kan du blive mere grøn ved at:
+            <br />- <b>Flytte</b> dit forbrug <br />- <b>Producere</b> din egen
+            energi <br />- Bruge <b>mindre</b> energi <br />- Købe grønne
+            <b>certifikater</b>
+          </p>
+          <p>
+            Flere af disse muligheder kræver, at strømmen er produceret samtidig
+            med, at den bliver forbrugt. Denne
+            <a routerLink="../{{ routes.simultaneity }}">samtidighed</a> kan du
+            læse mere om, da det er en af de væsentlige ting i det nye
+            certifikat.
+          </p>
+        </eo-stack>
+      </eo-info-box>
+      <div class="case">
+        <img alt="Iværksætter case" src="/assets/images/help/ivan_case.jpg" />
+        <div class="text">
+          <h4><b>CASE</b></h4>
```

Comment:
I would put the uppercase and font-weight into .case h4 css.

Reply:
Yeaaah, thats actually a pretty good idea :)